### PR TITLE
java-spring

### PR DESCRIPTION
**Pull request by lewinkedrs**
*Description of changes:*
Adds article about manual instrumentation of java-spring applications.

### CODE REVIEW

File: docs/en/guides/operational/adot-at-scale/adot-java-spring/adot-java-spring.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+Given that one of the main benefits of tracing is end-to-end visibility of a request, it is important for traces to link properly all the way from the request origin to the backend. A common way of doing this in OpenTelemetry is to utilize [nested spans](https://opentelemetry.io/docs/instrumentation/java/manual/#create-nested-spans). This works in a microservices architecture where the spans are passed from service to service until they reach the final destination. In a Spring application, we need to create parent/child relationships between spans created both remotely AND locally.
+
+## Tracing Utilizing Context Propagation
+
+This time we are going to suggest a different approach: Instead of passing spans around the application code, we are going to use the messages in the application to transport the context from spans associated with the previous message. To achieve this we are going to use this [approach](https://opentelemetry.io/docs/instrumentation/java/manual/#context-propagation).
```
Comment (rapphil) [inline docs/en/guides/operational/adot-at-scale/adot-java-spring/adot-java-spring.md:20]: This paragraph fells disconnected from the rest.

Comment (rapphil) [reply docs/en/guides/operational/adot-at-scale/adot-java-spring/adot-java-spring.md:20]: I think you can direct to the point and say that we are going to use context propagation to manually instrument the application

File: mkdocs.yaml [hunk 2]
```diff
@@ -49,7 +49,8 @@ nav:
 
     - Operational:
       - ADOT Collector:
-        - Operating at scale: guides/operational/adot-at-scale/operating-adot-collector.md
+        - Operating At Scale: guides/operational/adot-at-scale/operating-adot-collector.md
+        - Manually Instrumenting Spring Applications: guides/operational/adot-at-scale/adot-java-spring/adot-java-spring.md
```
Comment (rapphil) [inline mkdocs.yaml:53]: Sprint integration application?

**Review by rapphil — CHANGES_REQUESTED** [review]
Please address the comments.
We need to make sure that all references to spring are to Spring Integration.

also make sure that we are targeting the broader audience instead of the audience from where this case was extracted.

**Review by rapphil — APPROVED** [review]

**Review by awsimaya — APPROVED** [review]
LGTM

**Pull request merged**